ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -1785,3 +1785,64 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    job-type: launch
+    job-env: openstack
+    job-release: '4.2'
+  interval: 4h
+  name: canary-openshift-ocp-installer-e2e-openstack-4.2
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/e2e-openstack-cluster-profile
+      - --target=e2e-openstack
+      - --template=/usr/local/e2e-openstack
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.2
+      - name: BRANCH
+        value: "4.2"
+      - name: CLUSTER_TYPE
+        value: openstack
+      - name: CONFIG_SPEC
+        value: |
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+      - name: JOB_NAME_SAFE
+        value: e2e-openstack
+      - name: KURYR_ENABLED
+        value: "false"
+      - name: TEST_COMMAND
+        value: run-minimal-tests
```

Comment:
How long until this is fixed to run all e2e tests?

Reply:
Following up on your question about tests this afternoon. The answer is I think we can try them now. The reason we ran minimal for some of the openstack pre/post submit jobs I think was solely to save time on those jobs.